src/Files.App/Strings/en-US/Resources.resw

```diff
@@ -3526,7 +3370,138 @@
   <data name="ClearCompleted" xml:space="preserve">
     <value>Clear completed</value>
   </data>
+  <data name="NameWithColon" xml:space="preserve">
+    <value>Name:</value>
```

Comment:
Can you check if there is already a similar resource?

Reply:
There is. ToolTipDescriptionName.

Do you'd like me to rename ToolTipDescriptionName with NameWithColon to avoid duplication?

Reply:
Done